examples/src/main/scala/org/apache/spark/examples/mllib/PowerIterationClusteringExample.scala

```diff
@@ -18,141 +18,42 @@
 // scalastyle:off println
 package org.apache.spark.examples.mllib
 
-import org.apache.log4j.{Level, Logger}
-import scopt.OptionParser
-
 import org.apache.spark.{SparkConf, SparkContext}
-import org.apache.spark.mllib.clustering.PowerIterationClustering
-import org.apache.spark.rdd.RDD
+// $example on$
+import org.apache.spark.mllib.clustering.{PowerIterationClustering, PowerIterationClusteringModel}
+// $example off$
 
-/**
```

Comment:
@yinxusen could you please explain more how to reuse? 
previous examples are quite different from what is shown insdie {highlight} block